src/handbook/design/process.md

```diff
@@ -16,7 +18,19 @@ been conducted on the high-level UX, and development is not blocked at the start
 of a release sprint. Both visual and UX design iterations will occur throughout the
 first two weeks of a cycle, alongside development efforts.
 
-### Delivery of Art
+### Website and rest of design assets
+
+In contrast to the platform, changes and updates to the web design can be implemented independently and are not limited to specific release cycles. The timing and order of these changes, as well as the creation of other design assets such as blog images, open graphs, presentations, illustrations, etc., depend on prioritization aligned with the company's interests.
+
+## Reviewing
+
+During this phase, it is essential, whenever possible, to thoroughly review the proposed content before proceeding with the implementation in code, particularly regarding UX/UI design.
```

Comment:
This is a tricky one, our Company value is iteration. Should we be blocking anyone from adding new pages to our website (e.g. Roadmap) when we haven't fully fleshed out a design. I'd say we shouldn't. 

The Roadmap is a great example of a page where @MarianRaphael went ahead and did a great first iteration with 0 design input, and we follow up with an improved UX/Design.

Reply:
This part is meant to address the content review, not the design, but I guess it's not clear enough.

Reply:
How about this?
```suggestion
During this phase, it is encouraged to review the proposed content before proceeding with the implementation in code, particularly regarding UX/UI design. While it is ideal to have a thorough review whenever possible, we also value the iterative nature of our company. 
```

Reply:
My understanding for Figma, wireframes, etc etc are mostly to align the ideas for how to proceed with features. As the implementation might surface new issues (browser support for needed features, ideas don't work for different screens, etc. etc) I think a quick review of Figma is great, but enough.

```suggestion
To gather feedback as early as possible the design phase as a quick review of the proposed content before proceeding with the implementation in code, particularly regarding UX/UI design.
```

Reply:
Responsive design (different screen sizes) should always be considered at the Figma level. If we're getting to the point of developing something, and _then_ realising the design doesn't work at a given screensize, something is wrong with that design process.

As for feature support, I don't think we're going to be pushing too many boundaries on a marketing website in that regard, so definitely not an area of concern